README.md

```diff
@@ -236,479 +236,900 @@ The base class sets defaults used by other defined types, such as `tomcat::insta
+Specifies which Service element the Engine should nest under.
+
+Valid options: a string containing the name attribute of the Service.
+
+Default value: `Catalina`.
 
 #####`server_config`
 
-Specifies a server.xml file to manage. Valid options: a string containing an absolute path. Default: '${catalina_base}/config/server.xml'.
+Specifies a server.xml file to manage.
+
+Valid options: a string containing an absolute path.
+
+Default value: `${catalina_base}/config/server.xml`.
 
 #####`start_stop_threads`
 
-Sets how many threads the Engine should use to start child Host elements in parallel. Maps to the [startStopThreads XML attribute](http://tomcat.apache.org/tomcat-8.0-doc/config/engine.html#Common_Attributes). Valid options: a string. Default: undef.
+Sets how many threads the Engine should use to start child Host elements in parallel. Maps to the [startStopThreads XML attribute](http://tomcat.apache.org/tomcat-8.0-doc/config/engine.html#Common_Attributes).
+
+Valid options: a string.
+
+Default value: `undef`.
 
 #####`start_stop_threads_ensure`
 
-Specifies whether the [startStopThreads XML attribute](http://tomcat.apache.org/tomcat-8.0-doc/config/engine.html#Common_Attributes) should exist in the configuration file. Valid options: 'true', 'false', 'present', and 'absent'. Default: 'present'.
+Specifies whether the [startStopThreads XML attribute](http://tomcat.apache.org/tomcat-8.0-doc/config/engine.html#Common_Attributes) should exist in the configuration file.
+
+Valid options: `true`, `false`, `present`, and `absent`.
+
+Default value: `present`.
 
 #### tomcat::config::server::globalnamingresource
 
-Configure GlobalNamingResources Resource elements in $CATALINA\_BASE/conf/server.xml
+Configure GlobalNamingResources Resource elements in `$CATALINA_BASE/conf/server.xml`
 
 ##### `ensure`
 
-Determines whether the specified XML element should exist in the configuration file. Valid options: 'true', 'false', 'present', and 'absent'. Default: 'present'.
+Determines whether the specified XML element should exist in the configuration file.
+
+Valid options: `true`, `false`, `present`, and `absent`.
+
+Default value: `present`.
 
 ##### `resource_name`
 
 Optionally override the globalnamingresource name that is normally taken from the Puppet resource's `$name`.
 
 ##### `catalina_base`
 
-Specifies the base directory of the Tomcat instance. Valid options: a string containing an absolute path. Default: `$::tomcat::catalina_home`
+Specifies the base directory of the Tomcat instance.
+
+Valid options: a string containing an absolute path.
+
+Default value: `$::tomcat::catalina_home`
 
 ##### `type`
 
-Specifies the type of element to create. Defaults to `Resource` but can also be set to `Environment`, among any other valid node.
-_Note: This is used verbatim in your configuration so make sure the case is correct._
+Specifies the type of element to create
+
+Valid options: . `Resource`, `Environment` or any other valid node.
```

Comment:
I am not sure if the '.' is a value or a typo. These values are strings (even though we don't say that), so should be in single quotes.